core/src/trezor/wire/thp/received_message_handler.py

```diff
@@ -177,6 +177,9 @@ async def _handle_state_handshake(
             if paired:
                 trezor_state = _TREZOR_STATE_PAIRED
                 ctx.credential = credential
+                if ctx.is_channel_to_replace():
+                    # When replacing existing channel, user confirmation is not needed
+                    trezor_state = _TREZOR_STATE_PAIRED_AUTOCONNECT
```

Comment:
Will it also be sent to the client in case of an "auto-connect" credential?
If no - maybe it should be renamed?